Dockerfile

```diff
@@ -11,4 +11,4 @@ WORKDIR /app
 
 EXPOSE 8080
 
-ENTRYPOINT ["sh", "-c", "java ${JAVA_OPTS} -jar /app/ehrbase.jar --spring.profiles.active=docker"]
```

Comment:
Why?

Reply:
The problem was that this format did not accept arguments. But I forgot to support the `JAVA_OPTS` env var. I have extracted the command into an entrypoint script. This script now supports env vars and arguments.